robot-name/hemartin/robot_name.go

```diff
@@ -0,0 +1,54 @@
+package robotname
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+const letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const numberUpperLimit = 999
+
+var robotNumber = 0
+var robotNames = func() []string {
+	names := []string{}
```

Comment:
I just wonder if creating an array of length the max number of robots wouldn't be faster compared to using a slice and resizing the underlying array with each append.

Reply:
Agree, will be probably faster. Also I think it will start the program with the memory needed for this object already allocated, which is nice for constrained environments.